nemo/examples/nlp/language_modeling/llama_13b.sh

```diff
@@ -5,6 +5,7 @@ export TP=8
 export PP=4
 export N_LAYERS=40
 export N_AH=40
+export N_QG=40
```

Comment:
Can you post performance numbers and convergence curves for pretraining with your changes? Can you use below config

tensor_parallel:8
pipeline_parallel:8
data_parallel:1
global_batch_size:256
activation_checkpointing:full
precision:bf16+SR
dataset:bookcorpus
lrscheduler: cosineannealing

Reply:
I have confirmed that the number of seconds per step is reduced on 7B with a smaller num_query_groups setting than the normal setting.

We plan to start training on 70B next week.
We will share the results through our contact at AWS Japan.

In preparation, we plan to do some experiments with smaller settings.
I will share the results as soon as we are done.